common/test/java/com/couchbase/lite/BaseDbTest.java

```diff
@@ -55,6 +59,12 @@ public interface DocValidator extends Fn.ConsumerThrows<Document, CouchbaseLiteE
 
     public static C4Database getC4Db(@NonNull Database db) { return db.getOpenC4Database(); }
 
+    protected static <T extends Comparable<T>> void assertContents(List<T> l1, T... contents) {
+        List<T> l2 = Arrays.asList(contents);
```

Comment:
Huh.  I didn't know they had to be sorted to be considered equals.


This has the side effect of sorting the lists that are passed to it.  It would be better if it didn't do that... but at least it needs a comment.